lib/sender.js

```diff
@@ -93,8 +93,7 @@ function send(client) {
 function sendBatch(client) {
     return function sendBatch(messages) {
         const body = messages.map(m => composeSenderRequestBoddyForOneMessage(m.message, m.options));
-        client.url += 'batch';
-        return makeRequest(client, body);
+        return makeRequest(client, body, client.url + 'batch');
```

Comment:
@josemigallas should it end with a `/` ? 

like here: https://github.com/aerogear/aerogear-unifiedpush-nodejs-client/blob/master/lib/unifiedpush-node-sender.js#L53